plugins.json

```diff
@@ -670,5 +670,17 @@
         "download": "https://github.com/funktechno/kanboard-plugin-wiki/releases/download/0.2.5-alpha/Wiki-0.2.5.zip",
         "remote_install": true,
         "compatible_version": ">=1.0.37"
+    },
+    "blueboard": {
+        "title": "Blueboard",
+        "version": "0.0.1",
+        "author": "bgibout",
+        "licence": "MIT",
+        "description": "Simple blue CSS Theme",
+        "homepage": "https://github.com/bgibout/blueboard",
+        "readme": "https://raw.githubusercontent.com/bgibout/blueboard/master/README.md",
+        "download": "https://github.com/bgibout/blueboard/archive/master.zip",
```

Comment:
You should make your own zip file because the directory structure of GitHub archives is not the same as the one expected by Kanboard.

If you don't want to make your own archive, set to false "remote_install".